packages/forms/src/UIForm/fieldsets/Array/__snapshots__/Array.component.test.js.snap

```diff
@@ -6,7 +6,6 @@ exports[`Array component #renderItem should render item at given index 1`] = `
   errorMessage="This array is not correct"
   id="talend-array-1"
   isValid={true}
-  items={Array []}
```

Comment:
This is due to a defaultProps that was not used. I removed it, but with @talend/scripts jest version, the defaultProps appear in the snapshots